### PR TITLE
Update CHANGELOG for docker.io fix

### LINKED ISSUE #1566: Update CHANGELOG for docker.io support
As we've fixed #1549.

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #1566.

### CODE REVIEW

File: jib-gradle-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 - `os` and `architecture` are taken from base image ([#1564](https://github.com/GoogleContainerTools/jib/pull/1564))
 
 ### Fixed
+- Fixed an issue that pushing to Docker Hub fails when the host part of an image reference is `docker.io` ([#1549](https://github.com/GoogleContainerTools/jib/issues/1549))
```
Comment (TadCordle) [inline jib-gradle-plugin/CHANGELOG.md]: "Fixed an issue ~~that~~ where" might sound a bit better.

File: jib-maven-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 - `os` and `architecture` are taken from base image ([#1564](https://github.com/GoogleContainerTools/jib/pull/1564))
 
 ### Fixed
+- Fixed an issue that pushing to Docker Hub fails when the host part of an image reference is `docker.io` ([#1549](https://github.com/GoogleContainerTools/jib/issues/1549))
```
Comment (TadCordle) [inline jib-maven-plugin/CHANGELOG.md]: "Fixed an issue ~~that~~ where"

**Comment by TadCordle** [comment]
Oh, should we update the jib-core log as well? Or was this a plugin-side change? I don't remember.

**Comment by chanseokoh** [comment]
> Oh, should we update the jib-core log as well?

Ah, good catch. Will update that.

**Pull request merged**